src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderProgramProfileBuilder.kt

```diff
@@ -44,21 +31,7 @@ class OffenderProgramProfileBuilder(
       endComment = endComment,
     ).apply {
       payBands.addAll(this@OffenderProgramProfileBuilder.payBands.map { it.build(this) })
-    }
-
-  fun build(courseActivity: CourseActivity): OffenderProgramProfile =
```

Comment:
Allocations are no longer a child of the course activity DSL so not needed any more.